bitnami/harbor/values.yaml

```diff
@@ -3701,6 +3716,251 @@ trivy:
       http: 8080
       https: 8443
 
+## @section Harbor Exporter Parameters
+
+exporter:
+  ## Bitnami Harbor Exporter image
+  ## ref: https://hub.docker.com/r/bitnami/harbor-exporter/tags/
+  ## @param exporter.image.registry Registry for exporter image
+  ## @param exporter.image.repository Repository for exporter image
+  ## @param exporter.image.tag Tag for exporter image
+  ## @param exporter.image.pullPolicy Harbor exporter image pull policy
+  ## @param exporter.image.pullSecrets Specify docker-registry secret names as an array
+  ## @param exporter.image.debug Specify if debug logs should be enabled
+  ##
+  image:
+    registry: docker.io
+    repository: bitnami/harbor-exporter
+    tag: 2.4.1-debian-10-r0
+    ## Specify a imagePullPolicy
+    ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+    ## ref: https://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+    ##
+    pullPolicy: IfNotPresent
+    ## Optionally specify an array of imagePullSecrets.
+    ## Secrets must be manually created in the namespace.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ## e.g:
+    ## pullSecrets:
+    ##   - myRegistryKeySecretName
+    ##
+    pullSecrets: []
+    ## Set to true if you would like to see extra information on logs
+    ## It turns BASH and/or NAMI debugging in the image
+    ##
+    debug: false
+  ## @param exporter.command Override default container command (useful when using custom images)
+  ##
+  command: []
+  ## @param exporter.args Override default container args (useful when using custom images)
+  ##
+  args: []
+  ## @param exporter.extraEnvVars Array containing extra env vars
+  ## For example:
+  ##  - name: HARBOR_DATABASE_SSLMODE
+  ##    value: verify-ca
+  ##
+  extraEnvVars: []
+  ## @param exporter.extraEnvVarsCM ConfigMap containing extra env vars
+  ##
+  extraEnvVarsCM: ""
+  ## @param exporter.extraEnvVarsSecret Secret containing extra env vars (in case of sensitive data)
+  ##
+  extraEnvVarsSecret: ""
+  ## @param exporter.containerPorts.metrics Harbor Exporter HTTP container port
+  ##
+  containerPorts:
+    metrics: 8001
+  ## @param exporter.replicaCount The replica count
+  ##
+  replicaCount: 1
+    ## Harbor Exporter containers' liveness probe
+  ## ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes
+  ## @param exporter.livenessProbe.enabled Enable livenessProbe
+  ## @param exporter.livenessProbe.initialDelaySeconds Initial delay seconds for livenessProbe
+  ## @param exporter.livenessProbe.periodSeconds Period seconds for livenessProbe
+  ## @param exporter.livenessProbe.timeoutSeconds Timeout seconds for livenessProbe
+  ## @param exporter.livenessProbe.failureThreshold Failure threshold for livenessProbe
+  ## @param exporter.livenessProbe.successThreshold Success threshold for livenessProbe
+  ##
+  livenessProbe:
+    enabled: true
+    initialDelaySeconds: 20
```

Comment:
20 seconds seems like a very high value for "initialDelaySeconds". Why did you choose this value?

Reply:
In the upstream project the `intialDelaySeconds` was 300. I reduced and tested these values and they works fine. I've noticed that some times the  exporter spin up before the `jobservice` and in that case the set of metrics returned by the exporter is not complete for some seconds. Maybe is worth to include an init container to wait for the `jobservice`

Reply:
Fair enough